recipes/BEHST/meta.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: behst
+  version: 29efb4c3a8e333a2824430ef32d8bf01
+
+source:
+  fn: 326f13179c55.zip
+  url: https://bitbucket.org/hoffmanlab/behst/get/326f13179c55.zip
+  md5: 59b4e4ede8d1362ccc2de312d8f39421
+
+build:
+  number: 1
+  skip: True # [not py27]
+  skip: True # [osx]
+    
+requirements:
+  build:
+    - mercurial
+    - python
+    - pandas
+    - zlib
+    - libpng
+    - bedtools
+    - pybedtools
+    - r-base
+    - curl
+    - r-gprofiler
+  run:
+    - mercurial
+    - python
+    - pandas
+    - zlib
+    - libpng
+    - bedtools
+    - pybedtools
+    - r-base
+    - curl
+    - r-gprofiler
+    
+test:
+  requires:
```

Comment:
all the requirements are not needed, as these are your runtime dependencies
test-reqs are only useful if you need to have deps that are not in your runtime.